drivers/adc/ad7768-1/ad77681.c

```diff
@@ -245,10 +245,10 @@ uint8_t ad77681_get_rx_buf_len(struct ad77681_dev *dev)
  * @param dev - The device structure.
  * @return frame_16bit - the number of 16 bit SPI frames
  */
-uint8_t ad77681_get_frame_16bit(struct ad77681_dev *dev)
```

Comment:
Can you add in the commit message why this change is needed?

Reply:
Done adding comments